source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet

```diff
@@ -3,18 +3,49 @@ local rules = import 'rules.libsonnet';
 
 local listName = 'household';
 
-local question(title, description, additionalAnswerOptions=[]) = {
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'One year ago, what was <em>{person_name_possessive}</em> usual address?',
+    placeholders: [
+      placeholders.personNamePossessive,
+    ],
+  }
+  else 'One year ago, what was your usual address?'
+);
+
+local additionalAnswerOption(isFirstPerson) = (
+  if isFirstPerson then [{
+    label: {
+      text: 'Same as {first_person_possessive} address one year ago',
+      placeholders: [
+        placeholders.firstPersonNamePossessiveForList(listName),
+      ],
+    },
+    value: 'Same as {first_person_possessive} address one year ago',
+  }]
+  else []
```

Comment:
This is the wrong way round. If it is the first person, they shouldn't get the option to say the address is the same as the first person.